models/user.js

```diff
@@ -2,7 +2,9 @@
 
 const bcrypt = require('bcrypt');
 const mongoose = require('mongoose');
+const validator = require('validator');
```

Comment:
You have to be kiddin' me :upside_down_face: 
Bringing in library that has among all other stuff freakin' postal code validation in order to check email? You just killed a puppy :dog: :gun: